src/OrchardCore.Modules/OrchardCore.Lists/Feeds/ListFeedEditViewModel.cs

```diff
@@ -1,11 +1,15 @@
-﻿using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement;
 
 namespace OrchardCore.Lists.Feeds
 {
     public class ListFeedEditViewModel
     {
+        public bool DisableRssFeed { get; set; }
```

Comment:
It woud be better to use `DisableFeed` instead. Coz the feed could be ATOM feed

Reply:
But the view actually renders an RSS one
`Type = "application/rss+xml",` and it doesn't really matter